test/integration/routes/chatbot.test.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+// libs
+require('dotenv').config();
+const test = require('ava');
+const chai = require('chai');
+const sinonChai = require('sinon-chai');
+const request = require('supertest');
+const mongoose = require('mongoose');
+
+const stubs = require('../../utils/stubs');
+
+const chatbotRoutePath = '/v1/chatbot';
+
+// setup "x.should.y" assertion style
+chai.should();
+chai.use(sinonChai);
+
+// config
+require('../../../config');
+
+// app
+const app = require('../../../app');
+
+test.after.always(() => {
+  mongoose.connection.db.dropDatabase();
+});
+
+test('Should fail if request does not include a valid api key', async () => {
+  await request(app)
```

Comment:
It's an async function. Maybe it'll be better without using then?

Reply:
You're right! I'm going to use the [.expect](https://www.npmjs.com/package/supertest#expectstatus-fn) function instead.